@rajputanuj31 You can check the logs here https://ci.joomla.org/joomla/joomla-cms/72498/1/35 . Just use the "Show all logs" at the top of the dark frame at the right hand site to show the complete log, or use the download button in the top right corner to download the complete log file.

It shows that the "Test that menus site API endpoint can deliver a list of site menus" test is failing:
```
 Running:  api/com_menus/Site.cy.js                                                     (87 of 109)

  Test that menus site API endpoint
    1) can deliver a list of site menus
    ✓ can deliver a single site menu (335ms)
    ✓ can create a site menu (287ms)
    ✓ can update a site menu (429ms)
    ✓ can delete a site menu (276ms)

  4 passing (7s)
  1 failing

  1) Test that menus site API endpoint
       can deliver a list of site menus:
     AssertionError: Timed out retrying after 4000ms: expected 'Main Menu' to include 'automated test site menu'
      at  (webpack://joomla/./tests/System/integration/api/com_menus/Site.cy.js:9:9)
      at getRet (http://localhost/__cypress/runner/cypress_runner.js:118033:20)
      at tryCatcher (http://localhost/__cypress/runner/cypress_runner.js:1807:23)
      at __webpack_modules__</module.exports/Promise.try (http://localhost/__cypress/runner/cypress_runner.js:4315:29)
      at thenFn (http://localhost/__cypress/runner/cypress_runner.js:118044:60)
      at then (http://localhost/__cypress/runner/cypress_runner.js:118293:21)
      at wrapped (http://localhost/__cypress/runner/cypress_runner.js:138252:19)
      at __webpack_modules__</runCommand (http://localhost/__cypress/runner/cypress_runner.js:144143:15)
      at tryCatcher (http://localhost/__cypress/runner/cypress_runner.js:1807:23)
      at __webpack_modules__</module.exports/Promise.prototype._settlePromiseFromHandler (http://localhost/__cypress/runner/cypress_runner.js:1519:31)
      at __webpack_modules__</module.exports/Promise.prototype._settlePromise (http://localhost/__cypress/runner/cypress_runner.js:1576:18)
      at __webpack_modules__</module.exports/Promise.prototype._settlePromiseCtx (http://localhost/__cypress/runner/cypress_runner.js:1613:10)
      at _drainQueueStep (http://localhost/__cypress/runner/cypress_runner.js:2411:12)
      at _drainQueue (http://localhost/__cypress/runner/cypress_runner.js:2400:24)
      at __webpack_modules__</Async.prototype._drainQueues (http://localhost/__cypress/runner/cypress_runner.js:2416:16)
      at __webpack_modules__</Async/this.drainQueues (http://localhost/__cypress/runner/cypress_runner.js:2286:14)
```

Unfortunately there was (and maybe still is?) a problem with uploading the screenshots to the artifacts server so we don't have the screenshots from the failed test.

Does this help you a bit?

--- a/tests/System/integration/api/com_menus/Site.cy.js
+++ b/tests/System/integration/api/com_menus/Site.cy.js
@@ -4,9 +4,7 @@ describe('Test that menus site API endpoint', () => {
   it('can deliver a list of site menus', () => {
     cy.db_createMenuType({ title: 'automated test site menu' })
       .then(() => cy.api_get('/menus/site'))
-      .then((response) => cy.wrap(response).its('body').its('data.0').its('attributes')
-        .its('title')
-        .should('include', 'automated test site menu'));
+      .then((response) => cy.api_responseContains(response, 'title', 'automated test site menu'));
   });
 
   it('can deliver a single site menu', () => {

